oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -698,6 +698,11 @@ int mca_spml_ucx_ctx_create(long options, shmem_ctx_t *ctx)
         opal_progress_register(spml_ucx_ctx_progress);
     }
 
+    if (options & SHMEM_CTX_PRIVATE) {
```

Comment:
minor: could avoid code duplication if handle ` if (!(options & SHMEM_CTX_PRIVATE)) ` instead

Reply:
Good point, @janjust you can put 
https://github.com/open-mpi/ompi/pull/7065/files#diff-8af288f2560951b1798fcbf1131caa78L701-L703
into the if statement as 
https://github.com/open-mpi/ompi/pull/7065/files#diff-8af288f2560951b1798fcbf1131caa78R702-R703
and https://github.com/open-mpi/ompi/pull/7065/files#diff-8af288f2560951b1798fcbf1131caa78R710-R711
are essentially the same.